src/Hosting.Services.Web/Middlewares/UserIdentity/IdentityProviders/EmailBasedUserIdentityProvider.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares
+{
+	internal class EmailBasedUserIdentityProvider : IUserIdentityProvider
+	{
+        private static int MaxBytesPerChar { get; } = 4; // UTF-8
```

Comment:
These 2 constants are only used here and within unit tests, so I'd keep them as private constants to make them more efficient and avoid someone using them incorrectly:

```
private static const int MaxBytesPerChar = 4;
```

Reply:
Using const now

Reply:
You can do the same for `MaxBytesInIdentity`. I realise you are using this in a unit test, but it's not great practice to make this public just so it can be used in a unit test

Reply:
I missed that. Thanks for clarifying. It should be fine to expose it as a `const` without the getter by the way

Reply:
`EmailBasedUserIdentityProvider` is implementing `IUserIdentityProvider` where `public int MaxBytesInIdentity { get; }` is declared, I am not changing the interface there. Thus resolving this conversation.